torchvision/prototype/transforms/_geometry.py

```diff
@@ -136,30 +136,39 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         )
 
 
-class MultiCropResult(list):
-    """Helper class for :class:`~torchvision.prototype.transforms.BatchMultiCrop`.
-
-    Outputs of multi crop transforms such as :class:`~torchvision.prototype.transforms.FiveCrop` and
-    `:class:`~torchvision.prototype.transforms.TenCrop` should be wrapped in this in order to be batched correctly by
-    :class:`~torchvision.prototype.transforms.BatchMultiCrop`.
+class FiveCrop(Transform):
+    """
+    Example:
+        >>> class BatchMultiCrop(transforms.Transform):
+        ...     def forward(self, sample: Tuple[List[features.Image], features.Label]):
+        ...         images, labels = sample
+        ...         batch_size = len(images)
+        ...         images = features.Image.new_like(images[0], torch.stack(images))
+        ...         labels = features.Label.new_like(labels, labels.repeat(batch_size, *[1] * labels.ndim))
```

Comment:
If we stick with this minimal example, we could also remove this

```suggestion
        ...         labels = features.Label.new_like(labels, labels.repeat(batch_size))
```

since within this scope the label has a fixed shape, namely a scalar.

Reply:
Nice, let's lose this then.